src/attribute.ts

```diff
@@ -38,16 +38,21 @@ interface TypeElement {
 // computeType transforms a FHIR 'Element' into a single definitionId
 // that can be fetched from the FHIR API.
 // It may be a either:
+// - a primitive type indicated by the "structuredefinition-fhir-type" extension
 // - a basic resource/complex-type/primitive-type (element.code)
 // - a profile (element.profile[0])
-// - an extension (element.extension[0].url)
 const computeType = (element: TypeElement): string => {
-  if (element.extension && element.extension.length > 0) {
-    return (
-      element.extension[0].valueUrl ||
-      element.extension[0].url.split('/').pop()!
+  const primitive =
+    element.extension &&
+    element.extension.find(
```

Comment:
Any idea why this field is called extension in fhir?

Reply:
i believe it's when the attribute must have extensions with specific values